addons/context_actions/functions/fnc_canEditInventory.sqf

```diff
@@ -0,0 +1,20 @@
+#include "script_component.hpp"
+/*
+ * Author: mharis001
+ * Checks if the inventory of the given entity can be edited.
+ *
+ * Arguments:
+ * 0: Entity <ANY>
+ *
+ * Return Value:
+ * Can Edit Inventory <BOOL>
+ *
+ * Example:
+ * [_entity] call zen_context_actions_fnc_canEditInventory
+ *
+ * Public: No
+ */
+
+params ["_entity"];
+
+_entity isEqualType objNull && {alive _entity} && {getNumber (configFile >> "CfgVehicles" >> typeOf _entity >> "maximumLoad") > 0}
```

Comment:
Maybe push each and check into a new line? E.g.:
```suggestion
_entity isEqualType objNull
    && {alive _entity}
    && {getNumber (configFile >> "CfgVehicles" >> typeOf _entity >> "maximumLoad") > 0}
```

I'm not certain how the whole project manages them but I think these context_action functions would benefit from them to being more quickly readable without just being a long line of &&'s.

Reply:
No real rule, I generally multi-line if there are more than 3 or 4 conditions.